ethereum/gateway-eth-ts/src/GatewayTsBase.ts

```diff
@@ -70,28 +70,25 @@ export class GatewayTsBase {
       this.wallet || this.provider,
       addresses[this.networkId].flagsStorage
     );
-    gatewayTokenAddresses[this.networkId].forEach(
-      (gatewayToken: GatewayTokenItem) => {
-        const tokenAddress = gatewayToken.address;
-
-        if (
-          defaultGatewayToken !== null &&
-          tokenAddress === defaultGatewayToken
-        ) {
-          this.defaultGatewayToken = defaultGatewayToken;
-        }
-
-        this.gatewayTokens[tokenAddress] = {
-          name: gatewayToken.name,
-          symbol: gatewayToken.symbol,
-          address: gatewayToken.address,
-          tokenInstance: new GatewayToken(
-            this.wallet || this.provider,
-            gatewayToken.address
-          ),
-        };
+    for (const gatewayToken of gatewayTokenAddresses[this.networkId]) {
+      const tokenAddress: string = gatewayToken.address;
```

Comment:
Is it not easier to type `gatewayToken.address`? Are we ignoring an undefined case here?